imap_processing/cli.py

```diff
@@ -7,17 +7,18 @@
 
 Use
 ---
-    python run_processing.py <instrument> <data_level>
+    imap_cli --instrument <instrument> --level <data_level>
 """
 
 import argparse
 import sys
 from abc import ABC, abstractmethod
+from typing import Optional
 
 from imap_processing import instruments, processing_levels
 
 
-def _parse_args():
+def _parse_args(cli_args: list):
```

Comment:
Why do we need the `cli_args` here and in `main()`? I was pretty sure in my testing that there was not extra args passed into the script's main function call...

https://docs.python.org/3/library/argparse.html#parsing-arguments
> In a script, [parse_args()](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_args) will typically be called with no arguments, and the [ArgumentParser](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser) will automatically determine the command-line arguments from [sys.argv](https://docs.python.org/3/library/sys.html#sys.argv).